src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -71,3 +73,11 @@ def serialize_many(
             user=as_user,
             serializer=ProjectSerializer(),
         )
+
+    def get_teams_for_project(self, *, organization_id: int, project_id: int) -> List[RpcTeam]:
```

Comment:
I definitely worry about this.  Is it possible to simple get the existing `RpcOrganization` to contain all this info inline rather than N+1 queries for the teams for each project?

Reply:
@corps  I ended up removing this service method altogether.